### PR TITLE
Add first pass at code

### PR DESCRIPTION
**Pull request by megasmack**
- Add code
- Add bower.json, .editorconfig
- Update README

/cc @chris-erickson @rsenk330 @edyken77 
:four:

### CODE REVIEW

File: bootstrap-colequalizer.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            _.colReset(el);
+            var tallest = 0;
+            $('[class*=col-]',el).each(function (i,e) {
+                var testHeight = $(e).height();
+                if (testHeight > tallest) {
+                    tallest = testHeight;
+                }
+            });
+            $('[class*=col-]',el).height(tallest);
+        });
```
Comment (megasmack) [inline bootstrap-colequalizer.js:55]: This function compares all the heights of the elements and sets them to the tallest one.

File: bootstrap-colequalizer.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    };
+
+    ColEqualizer.prototype.colReset = function(el) {
+
+        $('[class*=col-]',el).height('auto');
+
+    };
```
Comment (megasmack) [inline bootstrap-colequalizer.js:63]: This function resets all the heights so they can go back to default or be re-written.

File: bootstrap-colequalizer.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+        // If set, minWidth shows/hides the nav based on the size of the browser
+        // If minWidth not set, nav will always show
+        if (768 <= viewWidth) {
+            _.colHeight(_.el);
+        } else {
+            _.colReset(_.el);
+        }
+
+    };
```
Comment (megasmack) [inline bootstrap-colequalizer.js:78]: Run the height function on window resize.

File: bootstrap-colequalizer.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+        $win.load(function () {
+            // Run function on window resize
+            $win.on('resize', function () {
+                _.resizeWindow();
+            });
+            // Run function the first time
+            _.resizeWindow();
+        });
+
```
Comment (megasmack) [inline bootstrap-colequalizer.js:93]: Everything runs on window load so that the heights can be properly calculated.

**Comment by rsenk330** [comment]
:+1:

**Pull request merged**